package.json

```
@@ -35,7 +35,8 @@
     "test:e2e": "npm run test -- --config __tests__/e2e/jest.e2e-config.js",
     "test:e2e:watch": "npm run test -- --config __tests__/e2e/jest.e2e-config.js --watchAll",
     "test:e2e:watch:inspect": "node --inspect ./node_modules/jest/bin/jest.js --config __tests__/e2e/jest.e2e-config.js --watchAll --testTimeout 999999",
-    "check-updates": "npx npm-check-updates -u"
+    "check-updates": "npx npm-check-updates -u",
+    "prepublish": "build"
```

```suggestion
    "prepublish": "npm run build"
```

I think this is why the tests are failing in the ci.

---

I will change this to `"prepublishOnly": "npm run build"` following the docu [here](https://docs.npmjs.com/cli/v10/using-npm/scripts)